### PR TITLE
Create and Delete Server Snapshot Actions

### PR DESCRIPTION
**Pull request by pgorgolew**

### CODE REVIEW

File: executor/src/main/java/org/k8loud/executor/action/openstack/CreateServerSnapshotAction.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public CreateServerSnapshotAction(Params params, OpenstackService openstackService) throws ActionException {
+        super(params, openstackService);
+    }
+
+    @Override
+    public void unpackParams(Params params) {
+        region = params.getRequiredParam("region");
+        serverId = params.getRequiredParam("serverId");
+        snapshotName = params.getOptionalParam("snapshotName", null);
+        stopInstance = Boolean.parseBoolean(params.getOptionalParam("stopInstance", "false"));
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/action/openstack/CreateServerSnapshotAction.java:23]: Move parsing to the Params class

File: executor/src/main/java/org/k8loud/executor/action/openstack/DeleteServerSnapshotAction.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    public DeleteServerSnapshotAction(Params params, OpenstackService openstackService) throws ActionException {
+        super(params, openstackService);
+    }
+
+    @Override
+    public void unpackParams(Params params) {
+        region = params.getRequiredParam("region");
+        serverId = params.getRequiredParam("serverId");
+        keepOneSnapshot = Boolean.parseBoolean(params.getOptionalParam("keepOneSnapshot", "true"));
```
Comment (proman3419) [inline executor/src/main/java/org/k8loud/executor/action/openstack/DeleteServerSnapshotAction.java:21]: Move parsing to the Params class

**Review by proman3419 — CHANGES_REQUESTED** [review]

File: executor/src/main/java/org/k8loud/executor/action/openstack/CreateServerSnapshotAction.java (hunk 1 above)
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/action/openstack/CreateServerSnapshotAction.java]: done

File: executor/src/main/java/org/k8loud/executor/action/openstack/DeleteServerSnapshotAction.java (hunk 2 above)
Comment (pgorgolew) [reply executor/src/main/java/org/k8loud/executor/action/openstack/DeleteServerSnapshotAction.java]: done

**Review by proman3419 — APPROVED** [review]

**Pull request merged**